src/htmlTransformers.js

```diff
@@ -60,18 +60,20 @@ export const transformAsides = content => {
   });
 };
 
-export const transformRelatedContent = (content, lang) => {
+export const transformRelatedContent = (content, lang, isOembedRequest) => {
   content('div').each((_, div) => {
     const isRelatedContentGroup =
       div.attribs && div.attribs['data-type'] === 'related-content';
     if (isRelatedContentGroup) {
-      const relatedArticleList = createRelatedArticleList(
-        { locale: lang, articleCount: content(div).children().length },
-        content(div)
-          .children()
-          .toString()
-      );
-      content(div).before(relatedArticleList);
+      if (!isOembedRequest) {
```

Comment:
I think it would be better to do this check in relatedContentPlugin. With this change we still do a lot of unnecessary api requests.